setup.py

```diff
@@ -121,15 +121,14 @@ def is_requirement(line):
             'log-sensitive = edx_django_utils.logging.internal.log_sensitive:cli',
         ],
     },
-    license="AGPL 3.0",
```

Comment:
Apparently this field is only intended for use when you don't have a license with a known classifier, which we do in fact supply farther down.